.circleci/config.yml

```
@@ -24,4 +64,5 @@ workflows:
       - maven/test:
           executor: jdk
       - build:
-          context: SonarCloud
\ No newline at end of file
+          context: SonarCloud
+      - deploy-k8s
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)